Red 9163 React-18 Peer Dependency : Fix

prembalaji39 opened the pull request:
Peer Dependency Fix

Prem3997 commented:
- updated react-fusioncharts version from 3.1.2 to 4.0.0
- updated fusioncharts version from 3.13.5-sr.1 to 3.18
- updated react, react-dom versions to 18.0.2
- updated peer dependency range for react "0.14.0 - 18.x"

prembalaji39 commented:
- updated react-fusioncharts version from 3.1.2 to 4.0.0
- updated fusioncharts version from 3.13.5-sr.1 to 3.18
- updated react, react-dom versions to 18.0.2
- updated peer dependency range for react "0.14.0 - 18.x"
